### PR TITLE
data-lake-file-containers-hdlfscli

### PR DESCRIPTION
**Pull request by rnagweka**
Some additional changes were done to the existing tutorial so that the content becomes even more better to understand with respect to adding the client.crt and some changes to the hdlfscli commands.

### CODE REVIEW

**Comment by MichaelCzcz** [comment]
@rnagweka Please link your Github account to Open Source Portal as described here: https://jam4.sapjam.com/groups/A7NmsKmoG9vVaxizKhdQOk/overview_page/jpJF7KXn5B2tFFQVOQ1Ybm?edit_mode=true

File: tutorials/data-lake-file-containers-hdlfscli/data-lake-file-containers-hdlfscli.md [hunk 1]
```diff
@@ -102,7 +102,9 @@ Some HDLFSCLI help documentation should appear if it is successfully installed a
...
+To connect the HDLFSCLI to a HANA Data Lake file container, a certificate will need to be generated to make a secure connection. Below are the steps required to create a self-signed certificate to get started using the HDLFSCLI. You will require an installation of OpenSSL. Use your preferred Linux package installer to install OpenSSL if it is not already installed. If you're using a Windows machine, then Windows Subsystem Linux will have OpenSSL installed. Alternatively, OpenSSL can be installed for Windows. OpenSSL for Windows can be downloaded from [Here](https://slproweb.com/products/Win32OpenSSL.html).  
+
+Then, follow these steps to creating your self-signed certificate.
 
 **Note**: Make sure the certificate fields are not all exactly the same between the Certificate Authority (CA) and client certificates, otherwise it is assumed to be a self-signed cert and the cert validation below will fail.
```
Comment (MichaelCzcz) [inline tutorials/data-lake-file-containers-hdlfscli/data-lake-file-containers-hdlfscli.md:109]: Please adjust formatting as shown in the style guide. Don't start a note with the label "Note". Please do this for all notes with wrong formatting.
https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9#mcetoc_1dtqfqchq21

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by MichaelCzcz** [comment]
@rnagweka Can you please address the requested changes? Thank you.

**Comment by MichaelCzcz** [comment]
The notes still have the wrong formatting. Please format them as shown in the style guide. https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9#mcetoc_1dtqfqchq21

**Comment by rnagweka** [comment]
I have pushed the tutorial to production 
Please let me know if it's fine.

**Comment by MichaelCzcz** [comment]
How to format notes:

Wrong: 
Note: This is a note.

Correct:
>This is a note

**Comment by MichaelCzcz** [comment]
@rnagweka What's the status?

**Comment by MichaelCzcz** [comment]
@rnagweka I haven't heard from you in over a month. If you are still interested in this pull request, please reply by the end of this week. Otherwise I will close it.

**Comment by rnagweka** [comment]
Hey,
 Yes, I have done the necessary changes for the tutorial. Sorry had to replace my laptop in the past month and because of that I didnt have any data saved for this tutorial. Please could you review the changes and let me know

**Comment by MichaelCzcz** [comment]
Please sign CLA.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please address the comments and check for more typos. The spell checker is not perfect and may have missed some.

File: tutorials/data-lake-file-containers-hdlfscli/data-lake-file-containers-hdlfscli.md [hunk 2]
```diff
@@ -22,11 +22,11 @@ primary_tag: software-product-function>sap-hana-cloud\,-data-lake
 
 [ACCORDION-BEGIN [Step 1: ](Download the SAP HANA data lake Client)]
 
-The HDLFSCLI is included in the HANA Data Lake Client download from the [SAP software center](https://launchpad.support.sap.com/#/softwarecenter/search/HANA%2520Data%2520lake%2520client). The first step is to download and install the latest version of the HANA data lake client.
+The HDLFSCLI is included in the HANA Data Lake Client download from the [SAP software center](https://launchpad.support.sap.com/#/softwarecenter/template/products/_APP=00200682500000001943&_EVENT=NEXT&HEADER=Y&FUNCTIONBAR=Y&EVENT=TREE&NE=NAVIGATE&ENR=73555000100800003274&V=MAINT&TA=ACTUAL/HANA%20CLOUD%20CLIENTS). The first step is to download and install the latest version of the HANA data lake client.
```
[the remark this answers is not in the document]
Comment (rnagweka) [reply tutorials/data-lake-file-containers-hdlfscli/data-lake-file-containers-hdlfscli.md:25]: Sure, thanks for pointing it out. I will make the changes.

**Comment by rnagweka** [comment]
I have made the necessary changes to this tutorial. Please have a look. Thanks.

**Comment by MichaelCzcz** [comment]
Thanks. Please sign CLA and update your Github profile/git settings. 

image

**Comment by rnagweka** [comment]
image

 I have already signed the CLA a multiple times. Dont know why isnt it reflecting on your end. Let me know what can I do?

**Pull request merged**